README.md

```diff
@@ -94,20 +94,23 @@ $ helm plugin install https://github.com/codacy/helm-ssm/releases/download/lates
 ### Developer (From Source) Install
 
 If you would like to handle the build yourself, instead of fetching a binary,
-this is how recommend doing it.
+this is how we recommend doing it.
 
-First, set up your environment:
+- Make sure you have [Go](http://golang.org) installed.
 
-- You need to have [Go](http://golang.org) installed. Make sure to set `$GOPATH`
-
-Clone this repo into your `$GOPATH`. You can use `go get -d github.com/codacy/helm-ssm`
-for that.
+- Clone this project
 
+- In the project directory run
 ```sh
-$ cd $GOPATH/src/github.com/codacy/helm-ssm
 $ make install
 ```
 
+### Notice about using this plugin with Helm 2
+
+Since helm2 is deprecated the current version of the plugin only supports helm3. The last version
+to support helm2 is [v2.2.1](https://github.com/codacy/helm-ssm/releases/tag/2.2.1). There will be
+no further patches or updates to this legacy version.
+
```

Comment:
Maybe I would just reference this section at the very top of the readme file with a nice fat [DEPRECATED] kind of warning. Very much like what the stable helm chart repo has for the upstream repositories.

Reply:
I like the idea of making it more visible. Maybe not on top, because it makes it look like the plugin itself is deprecated. This is the case with the helm charts, and I'd argue that's why they chose to make it the topmost thing on the readme.

Reply:
I've moved it up top, but made it a bit less flashy than the chart deprecation noticies. I think it should be clear enough. wdyt?

Reply:
lgtm